std/assembly/typedarray.ts

```diff
@@ -1260,3 +1304,14 @@ function REVERSE<TArray extends ArrayBufferView, T>(array: TArray): TArray {
   }
   return array;
 }
+
+// @ts-ignore: decorator
+@inline
+function WRAP<TArray extends ArrayBufferView, T>(buffer: ArrayBuffer, byteOffset: i32 = 0, byteLength: i32 = buffer.byteLength): TArray {
```

Comment:
In general I think the `.wrap` API should be identical to what a user would otherwise expect of a `new TypedArray(buffer [, byteOffset [, length]])` call. Seems that I got this wrong in my initial comment and the original API actually takes a `length` argument (of the final view) instead of a `byteLength` (of the wrapped buffer).

Reply:
yes, it should be `length` not `byteLength`

Reply:
fixed